Update securing_applications guide for latest adapter changes (community) (#20995)
keycloak/keycloak: docs/documentation/securing_apps/topics/oidc/java/jboss-adapter.adoc

endif::[]

ifeval::[{project_community}==true]

> WARNING: We do not regularly test and maintain WildFly/EAP adapters. They may work only with WildFly version of 23 or earlier or with JBoss EAP 7. Also note

Reviewer: ```suggestion
WARNING: We do not regularly test and maintain WildFly/EAP adapters. They may work only with WildFly version 23 or earlier or with JBoss EAP 7. Also note
```

> that they may work only with JEE applications, but not with Jakarta applications. It is recommended to switch to Elytron OIDC Java adapter to
> secure your applications. It has very similar configuration to Keycloak Java adapters and migrating your applications to it should be smooth.

Reviewer: ```suggestion
that they may work only with JEE applications, but not with Jakarta applications. We recommend that you switch to Elytron OIDC Java adapter to
secure your applications. This adapter has a similar configuration to Keycloak Java adapters and migrating your applications to it should be smooth.
```
